src/manager_tests.cpp

```diff
@@ -23,154 +23,176 @@ struct Health
 
```

Comment:
I tabbified this document as it was all jacked up in GitHub code review, the test i added is at the very bottom.